src/main/app/package.json

```diff
@@ -5,10 +5,10 @@
   "proxy": "http://localhost:8080",
   "dependencies": {
     "@material-ui/core": "^4.11.2",
+    "@material-ui/icons": "^4.11.2",
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.3",
     "@testing-library/user-event": "^12.6.0",
-    "bootstrap": "^4.6.0",
     "react": "^17.0.1",
     "react-bootstrap": "^1.4.3",
```

Comment:
@JustinLohKingWei can you also remove `react-bootstrap` as a dependency if we are going to be using mainly bootstrap

Reply:
Working on it. Removed bootstrap via command

Reply:
Yes use: `yarn remove react-bootstrap`

Reply:
Alright everything checks out.